src/components/NormalLeftRow/index.jsx

```diff
@@ -39,8 +39,9 @@ function NormalLeftRow({ schema, classes }) {
 
 NormalLeftRow.propTypes = {
   schema: shape({
-    type: string.isRequired,
+    type: string,
     name: string,
+    close: string,
```

Comment:
A comment for each prop would be helpful. It doesn't have to be long. One liners are fine.

Reply:
Added comments!